introduction.md

```diff
@@ -1,2 +1,6 @@
 Hi, I'm Becky and my favorite Code School Path is JavaScript
 Code School Path: JavaScript
+Deadly Skills
+* JavaScript
+* Git
+* Node.js
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?